apps/wallet/src/ui/app/pages/home/tokens/TokenIconLink.tsx

```diff
@@ -19,10 +19,13 @@ import { trackEvent } from '_src/shared/plausible';
 
 export function TokenIconLink({
     accountAddress,
+    noSuiToken,
 }: {
     accountAddress: SuiAddress;
+    noSuiToken?: boolean;
 }) {
-    const stakingEnabled = useFeature(FEATURES.STAKING_ENABLED).on;
+    const stakingEnabled =
+        useFeature(FEATURES.STAKING_ENABLED).on && !noSuiToken;
```

Comment:
This should be removed, you could have no sui but have delegations you've created, and still want to view. Just revert this change

Reply:
Ok.